### PR TITLE
doc/instances: add UI instructions

### PR DESCRIPTION
**Pull request by ru-fu**
Add instructions for how to manage and configure instances through the UI.
This is part of a PoC for how we can present instructional content for different clients.

### CODE REVIEW

**Review by edlerd — COMMENTED** [review]
Looking good, thank you. I added some suggestions. Only one thing is off: custom configuration is persisted by the UI. The limitation is around the changed and not persisted content of the YAML editor.

File: doc/howto/instances_configure.md [hunk 1]
```diff
@@ -188,5 +238,30 @@ See [`PUT /1.0/instances/{name}`](swagger:/instances/instance_put) for more info
...
+```
+````
+
+````{group-tab} UI
+Instead of using the UI forms to configure your instance, you can choose to edit the YAML configuration of the instance.
+You must use this method if you need to update any configurations that are not available in the UI.
+
+```{important}
+If you edit the YAML configuration to include any fields that are not supported by the UI, you must do all further configuration of the instance through the YAML configuration.
+You must not use any UI forms to do configuration updates afterwards, even for configuration that is covered by the UI.
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/howto/instances_configure.md:250]: OK - with "persisted" you mean saved?

[the remark this answers is not in the document]
Comment (edlerd) [reply doc/howto/instances_configure.md:250]: Yes, once the custom config is saved it should not be removed when editing the instance through the UI.

File: doc/howto/instances_configure.md [hunk 2]
```diff
@@ -13,37 +13,60 @@ To store and reuse different instance configurations, use {ref}`profiles <profil
...
 
 ````{tabs}
 ```{group-tab} CLI
+To set the memory limit to 128 MiB, enter the following command:
+
     lxc config set my-container limits.memory=128MiB
 ```
 
 ```{group-tab} API
+To set the memory limit to 128 MiB, send the following request:
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/howto/instances_configure.md:57]: Yes, let's just use the 8 GiB from your screenshot. :)

File: doc/howto/instances_manage.md [hunk 3]
```diff
@@ -67,6 +76,12 @@ Query the following endpoint to show detailed information about an instance:
 
 See [`GET /1.0/instances/{name}`](swagger:/instances/instance_get) for more information.
 ```
+
+```{group-tab} UI
+Clicking an instance line in the overview will show a summary of the instance information.
+
+Click the instance name to see detailed information.
```
[the remark this answers is not in the document]
Comment (ru-fu) [reply doc/howto/instances_manage.md:83]: Yes, always good to be consistent. :)

**Review by edlerd — APPROVED** [review]
LGTM, thanks for the changes - everything reads very good and correct now. Just a tiny change request below.

File: doc/howto/instances_manage.md [hunk 4]
```diff
@@ -128,16 +152,32 @@ To stop an instance, send a PUT request to change the instance state:
...
+You are then prompted to confirm.
+
+<!-- Include start skip confirmation -->
+```{tip}
+To skip the confirmation prompt, hold the {kbd}`Shift` key while clicking.
+```
+<!-- Include end skip confirmation -->
+
+You can choose to force-stop the instance.
+If you don't and stopping the instance takes a long time, click the spinning stop button to go back to the confirmation prompt, where you can select to force-stop the instance.
```
[the remark this answers is not in the document]
Comment (tomponline) [reply doc/howto/instances_manage.md:168]: > If stopping the instance takes a long time (or the guest is not responding to the stop request) ...

**Review by tomponline — APPROVED** [review]

**Pull request merged**